src/BeamAdapter/component/mapping/AdaptiveBeamMapping.inl

```diff
@@ -45,6 +45,20 @@
 #include <sofa/core/ConstraintParams.h>
 #include <sofa/core/MechanicalParams.h>
 
+// execution policies only supported by MSVC >=2019 and GCC >=10
+#ifdef _MSC_VER
+    #define HAS_SUPPORT_STL_PARALLELISM (_MSC_VER > 1921)
```

Comment:
According to https://en.cppreference.com/w/cpp/compiler_support/17#C.2B.2B17_library_features, it is 1914

Reply:
I put the first (official) version of MSVC2019, as SOFA officially does not support MSVC2017 anymore (but I can change the version anyway)